lsp-mode.el

```diff
@@ -480,32 +480,32 @@ the server has requested that."
 ;;;###autoload(put 'lsp-enable-file-watchers 'safe-local-variable #'booleanp)
 
 (defcustom lsp-file-watch-ignored '(; SCM tools
```

Comment:
this changes is not related.

Reply:
Yup, layered commits. All of my current lsp-mode PRs have this issue, except the first one.

Reply:
Fixed it.